CONTRIBUTING.md

```diff
@@ -1,7 +1,34 @@
 # CONTRIBUTING
+Here you can find informations on how you could contribute to the project
 
-# Repository changes
-- 🔨 `:hammer:` : Makefile
+## Compile the project
+To compile the project, you must have CMake installed on your machine and then launch the build script corresponding to your operating system :
+
+#### Windows
+```powershell
+.\windows_build_rtype.bat
+```
+
+#### Linux
+```bash
+./build_rtype.sh
+```
+
+You will then have either a `r-type_server` and a `r-type_client` for linux, or a `r-type_server.exe` and a `r-type_client.exe` for windows.
+
+## Pull requests
+If you want to contribute to the project, you should do so with a pull request.
+
+If you want your pull request to be validated, it sould respect the following conditions :
+- The description of the pull request **must** be self-sufficient and **must** explain what is implemented in it. You should also mention issues related to the pull request (if any) in the description (e.g. `close #23`).
+- The code **must** be documented with Doxygen-like comment. Every class, methods and functions must have its documentation to explain what it does.
```

Comment:
I think it might be clearer to use "All" instead of "Every", and use plural conjugation for all the verbs:

"All classes, methods and functions must have their documentation to explain what they do."

Reply:
Fixed